dist/js/bootstrap-colorpicker.js

```diff
@@ -8,23 +8,21 @@
  *
  */
 
-(function(factory) {
-  var global = (typeof window === 'undefined') ? this : window;
-
-  if (typeof exports === 'object') {
-    module.exports = factory(global.jQuery, global);
-  } else if (typeof define === 'function' && define.amd) {
-    define(['jquery'], function(jq) {
-      return factory(jq, global);
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module unless amdModuleId is set
+    define(["jquery"], function(a0) {
+      return (factory(a0));
     });
-  } else if (global.jQuery && !global.jQuery.fn.colorpicker) {
-    factory(global.jQuery, global);
+  } else if (typeof exports === 'object') {
+    // Node. Does not work with strict CommonJS, but
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory(require("jquery"));
+  } else {
```

Comment:
why did you delete this part of the condition `else if (!jQuery.fn.colorpicker)` ?
it avoids re-executing the factory if the colorpicker plugin is already loaded.
can you restore this condition?